packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -64,7 +70,8 @@ export class ScGridTooltip {
                     <div class="awsui-util-label">Latest value:</div>
                     <div>
                       <strong style={{ color }}>
-                        <Value value={this.propertyPoint.y} />
+                        {icon && <sc-chart-icon name={icon} color={color} style={{ marginRight: '3px' }} />}
+                        <Value value={value + unit} />
                       </strong>{' '}
                       at{' '}
```

Comment:
avoid hard coded phrases, utilize `messageOverrides` instead

Reply:
Could I add properties to `messageOverrides ` object? The current attributes do not represent `at` and `Status`.

Reply:
yes we'll have to add more properties, I think what would be best is to create a `DialMessageOverrides` and add what is necessary.

Reply:
Done --- Add `DialMessageOverrides `.